src/watchdog.py

```diff
@@ -42,7 +42,8 @@ def _print_file_info(filename):
 
 def _send_mail(msg, to):
 	cfg = config.Config(CONFIG_FILENAME)
-	m = mail.Mail(cfg.email_user, cfg.email_password, cfg.email_smtp, cfg.email_sender_name)
+    m = mail.Mail(cfg.watchdog_email_user, cfg.watchdog_email_password,
```

Comment:
err, is the indentation here safe?

Reply:
It must be (it runs) but I will fix it! I had my editor configured wrong and created a whole lot of these...